drivers/filesystems/fastfat/finfo.c

```diff
@@ -864,6 +865,10 @@ VfatSetRenameInformation(
                              (vfatFCBIsDirectory(FCB) ?
                               FILE_NOTIFY_CHANGE_DIR_NAME : FILE_NOTIFY_CHANGE_FILE_NAME),
                              (DeletedTarget ? FILE_ACTION_REMOVED : FILE_ACTION_RENAMED_OLD_NAME));
+
+            if (IsDotOrDotDot(NewFile))
```

Comment:
You no longer need this since it was already checked above.

Reply:
Removed. Thanks.